api/src/main/java/org/openmrs/module/mirebalais/apploader/apps/patientregistration/SectionsPeru.java

```diff
@@ -18,11 +31,145 @@ public SectionsPeru(Config config) {
 
     @Override
     public void addSections(RegistrationAppConfig c) {
+        c.addSection(getIdentifierSection());
         c.addSection(getDemographicsSection());
         c.addSection(getContactInfoSection());
-        // c.addSection(getIdentifierSection());
         c.addSection(getSocialSection());
     }
+    @Override
+    public Section getIdentifierSection() {
+        Section s = new Section();
+        s.setId("patient-identification-section");
+        s.setLabel("registrationapp.patient.identifiers.label");
+        s.addQuestion(getDNIdocument());
+        s.addQuestion(getPassportdocument());
+        return s;
+    }
+
+    private Question getDNIdocument() {
+        Question q = new Question();
+        q.setId("national-id");
+        q.setLegend("zl.registration.patient.documenttype.dni.label");
+        q.setHeader("zl.registration.patient.documenttype.dni.label");
+
+        Field f = new Field();
+        f.setFormFieldName("patientIdentifier" + SesConfigConstants.PATIENTIDENTIFIERTYPE_DNI_UUID);
+        f.setUuid(SesConfigConstants.PATIENTIDENTIFIERTYPE_DNI_UUID);
+        f.setType("patientIdentifier");
+        f.setWidget(getTextFieldWidget(8));
+
+        q.addField(f);
+
+        return q;
+    }
+    private Question getPassportdocument() {
+        Question q = new Question();
+        q.setId("passport-id");
+        q.setLegend("zl.registration.patient.documenttype.passport.label");
+        q.setHeader("zl.registration.patient.documenttype.passport.label");
+
+        Field f = new Field();
+        f.setFormFieldName("patientIdentifier" + SesConfigConstants.PATIENTIDENTIFIERTYPE_PASSPORT_UUID);
+        f.setUuid(SesConfigConstants.PATIENTIDENTIFIERTYPE_PASSPORT_UUID);
+        f.setType("patientIdentifier");
+        f.setWidget(getTextFieldWidget(12));
+
+        q.addField(f);
+
+        return q;
+    }
+    public Section getContactInfoSection() {
+        Section s = new Section();
+        s.setId("contactInfo");
+        s.setLabel("registrationapp.patient.contactInfo.label");
+        s.addQuestion(getAddressQuestion());
+        s.addQuestion(getTelephoneNumberQuestion());
+        s.addQuestion(getCellphoneNumberQuestion());
+        s.addQuestion(getEmailQuestion());
+        return s;
+    }
+
+    public Question getAddressQuestion() {
```

Comment:
If there's no change between this and the function it overrides (in `SectionsDefault.java`), you can just use the one from SectionsDefault (i.e., get rid of this one)

Reply:
I added two questions for this section(and maybe for SES since no other location uses these), that's why i copy the function.